symphony/app/fbcnms-projects/magmalte/app/components/Gateways.js

```diff
@@ -294,8 +294,16 @@ function GatewayRowItem(props: Props) {
     <TableRow>
       <TableCell>
         <DeviceStatusCircle
-          isGrey={!gateway.enodebRFTXOn}
-          isActive={gateway.enodebRFTXOn === gateway.enodebRFTXEnabled}
+          isGrey={gateway.isBackhaulDown}
+          isYellow={
+              !gateway.enodebRFTXOn ||
+              !gateway.enodebRFTXEnabled ||
+              !gateway.mmeConnected
```

Comment:
this conditional should be !(TX enabled) && (!mme connected || !tx on) - yellow should only show up for sites that aren't configured to transmit